packages/rx_bloc_cli/README.md

```diff
@@ -96,7 +96,7 @@ Out of the box, a Rx Bloc CLI created projects includes:
 
 ✅ [Design system][design_system_lnk] - A single place where you can define all your colors, typography, assets and more. Your app's Light and Dark mode are already configured
 
-✅ [Testing][testing_lnk] - With Unit and Golden Tests you know your app is working as intended
+✅ [Testing][testing_lnk] - With Unit and Golden Tests you know your app is working as intended. Please read the Rx Bloc CLI specific [Golden Tests documentation][golden_tests_documentation_lnk].
```

Comment:
the change log and the version in pubspec needs to be updated. make sure [rxBlocCliPackageVersion constant](https://github.com/Prime-Holding/rx_bloc/blob/develop/packages/rx_bloc_cli/lib/src/rx_bloc_cli_constants.dart#L2) is updated to reflect the current version

Reply:
Done